package.json

```diff
@@ -132,6 +132,8 @@
     "@typescript-eslint/parser": "^5.21.0",
     "@vscode/test-electron": "^2.1.3",
     "ajv": "^8.11.0",
+    "ajv-formats": "^2.1.1",
```

Comment:
not use where this library is being used, or needed

Reply:
i believe i needed it while validating parsers, will revert this one!